File: C++/Class/table.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "Enter number = ";
    int num;
    cin >> num;

    cout << "Table = " << endl;
    for (int i = 1; i <= 10; i++)
    {
        cout << num * i << endl;
    }

    return 0;
}

File: College/JAVA/EXP 1/Table.java
import java.util.Scanner;

public class Table {

    public static void main(String[] args) {
        int num, i;

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number = ");
        num = sc.nextInt();

        for (i = 1; i <= 10; i++) {
            System.out.println(num + " X " + i + " = " + i * num);
        }
    }
}